src/arb.rs

```diff
@@ -1,36 +1,16 @@
 use quickcheck::{Arbitrary, Gen};
-use rand::seq::SliceRandom;
+use rand::Rng;
 
-use crate::{Code, Multihash, MultihashCode, U64};
-
-const HASHES: [Code; 15] = [
-    Code::Sha2_256,
-    Code::Sha2_512,
-    Code::Sha3_224,
-    Code::Sha3_256,
-    Code::Sha3_384,
-    Code::Sha3_512,
-    Code::Keccak224,
-    Code::Keccak256,
-    Code::Keccak384,
-    Code::Keccak512,
-    Code::Blake2b256,
-    Code::Blake2b512,
-    Code::Blake2s128,
-    Code::Blake2s256,
-    Code::Blake3_256,
-];
+use crate::{Multihash, U64};
 
 /// Generates a random valid multihash.
-///
-/// This is done by encoding a random piece of data.
 impl Arbitrary for Multihash<U64> {
     fn arbitrary<G: Gen>(g: &mut G) -> Self {
-        let code = *HASHES.choose(g).unwrap();
-        let data: Vec<u8> = Arbitrary::arbitrary(g);
-        // encoding an actual random piece of data might be better than just choosing
-        // random numbers of the appropriate size, since some hash algos might produce
-        // a limited set of values
-        code.digest(&data)
+        let code = g.gen::<u64>();
+        // Maximum size is 64 byte due to the `U64` generic
+        let size = g.gen_range(0, 64);
+        let mut data = [0; 64];
+        g.fill_bytes(&mut data);
+        Multihash::wrap(code, &data[..size]).unwrap()
```

Comment:
What does it mean to have a code in the range from 0..2^64-1?

Reply:
You might want to do something where you have plausible code values most frequently, and then just occasionally add a code value from the full possible range.

Reply:
So what about changing this to something like I mentioned in my [other comment](https://github.com/multiformats/rust-multihash/pull/87#issuecomment-714404397):

> Or having a distribution like 50% 1-byte, 25% 2-byte, 12.5% 3-byte, 12.5% 4-byte?

I think I'll try that and push another commit.